_projects/2025/100528259/100528259.Rmd

```diff
@@ -0,0 +1,986 @@
+
+common_sports_men
+```
+
+
+Then, I do the same for women, because there is a lot of data noise at women's sports. Vast majority of women's sports in 2016 did not exist in 1960 Summer Olympics.
+
+```{r, echo=TRUE}
+sports_1960_women <- filtered_data |> 
+  filter(Year == 1960, Sex == "F") |> 
+  pull(Sport)
+
+sports_2016_women <- filtered_data |> 
+  filter(Year == 2016, Sex == "F") |> 
+  pull(Sport)
+
+```
+
+
+Let's find the intersection for women as well.
+
+```{r, echo=TRUE}
+
+common_sports_women <- intersect(sports_1960_women, sports_2016_women)
+
+common_sports_women
+
+```
+
+
+Keeping only the intersecting sports will help me ensure there is no data noise.
+
+```{r, echo=TRUE}
+olympics <- filtered_data |> 
+  filter(Year %in% c(1960, 2016),
+         (Sport %in% common_sports_men & Sex == "M") | 
+         (Sport %in% common_sports_women & Sex == "F"))
+```
+
+Now I will work on BMI values and categories. So let's create a BMI variable first.
+
+```{r, echo=TRUE}
+filtered_data_BMI <- olympics |> 
+  mutate(BMI = Weight / (Height / 100)^2)
+```
+
+I calculate average BMI values for each sport.
+
+```{r}
+avg_BMI <- filtered_data_BMI |> 
+  group_by(Sport, Year, Sex) |> 
+  summarize(avg_BMI = mean(BMI, na.rm = TRUE), .groups = 'drop')
+```
+
+
+For the three facets that I will create in my plot, I specify sports.
+
+```{r, echo=TRUE}
+avg_BMI <- avg_BMI |> 
+  mutate(Facet_Group = case_when(
+    Sport %in% c("Water Polo", "Wrestling", "Swimming", "Shooting", 
+                 "Weightlifting", "Gymnastics", "Equestrianism", "Canoeing", "Sailing", "Hockey") ~ "1",
+    Sport %in% c("Gymnastics", "Athletics", "Fencing", "Canoeing", 
+                 "Swimming", "Diving", "Equestrianism", "Sailing") ~ "2",
+  ))
+```
+
+Picking colours again! 
+
+```{r, echo=TRUE}
+
+retro_colours_2 <- c(
+  "Athletics" = "#00a046",       
+  "Basketball" = "#FF6F20",      
+  "Boxing" = "#fcab98",          
+  "Canoeing" = "#7fcdbb",        
+  "Cycling" = "#bcbddc",         
+  "Diving" = "#FF448C",          
+  "Gymnastics" = "#d8b365",      
+  "Hockey" = "#F1C40F",          
+  "Rowing" = "#10EA00",          
+  "Sailing" = "#4400AA",         
+  "Shooting" = "#D167BF",        
+  "Swimming" = "#0057A1",         
+  "Weightlifting" = "#7A9A01",   
+  "Wrestling" = "#80BFFF",       
+  "Water Polo" = "#636363"  
+)
+
+```
+
+
+### Onto the plot
+
+In this section, I define two character vectors for male and female Olympic sports, categorizing them into two groups for men and one for women. These sports are chosen on the basis of their presence in both Olympic Games. In men's sports, I manually picked those that will display little or no overlap.
+
+```{r, echo=TRUE}
+men_sports <- c("Athletics", "Canoeing", "Swimming", "Shooting", "Weightlifting", "Gymnastics", "Basketball", "Boxing", "Sailing", "Hockey", "Wrestling")
+women_sports <- c("Gymnastics", "Athletics", "Fencing", "Canoeing", "Swimming", "Diving", "Sailing")
```

Comment:
Please wrap these two lines a little adding newlines, because they go off the page.